cmd/operator-sdk/bundle/validate.go

```diff
@@ -132,33 +171,73 @@ func (c bundleValidateCmd) run() (err error) {
 		fields["container-tool"] = c.imageBuilder
 	}
 	logger := log.WithFields(fields)
-	if viper.GetBool(flags.VerboseOpt) {
-		log.SetLevel(log.DebugLevel)
+	if c.json {
+		// Log as JSON instead of the default ASCII formatter.
+		log.SetFormatter(&log.JSONFormatter{})
+	} else {
+		if viper.GetBool(flags.VerboseOpt) {
+			log.SetLevel(log.DebugLevel)
+		}
 	}
-
 	val := bundle.NewImageValidator(c.imageBuilder, logger)
-
 	// Pull image if a tag was passed.
 	if c.imageTag != "" {
-		logger.Info("Unpacked image layers")
+		msg := "Unpacked image layers"
+		if c.json {
+			jsonOutput.addLog(msg, nil)
+		} else {
+			logger.Info(msg)
+		}
 		err = val.PullBundleImage(c.imageTag, dir)
 		if err != nil {
-			logger.Fatalf("Error to unpacking image: %v", err)
+			msg := "error to unpacking image"
+			if c.json {
+				jsonOutput.addLog(msg, err)
+				jsonOutput.printFatalJSON()
+			}
+			logger.Fatalf(msg+": %v", err)
```

Comment:
I don't think we want to be logging things to the same place that we're writing the results.

IMO, if `-o json-alpha1` is present, one of two things happens
1. The command exits with successfully with code `0` and the ONLY output is the JSON for the validation results
2. Some precondition makes it impossible for the bundle validation to occur, so we exit with a non-zero error status and log the reason for the failure.

Reply:
We still need to consider log output by operator-registry libraries, so in addition to return codes: write logged errors (precondition failures) to stderr as JSON objects (use a plain `logrus.JSONFormatter`), and write validation output to stdout.

Reply:
we can use the log for all instead of implementing our output object. changed.